privaterelay/templates/includes/modal.html

```diff
@@ -6,8 +6,8 @@
   <div class="modal modal-content drop-shadow-light">
     <h2 class="modal-headline">{% ftlmsg 'modal-delete-headline' %}</h2>
     <span class="alias-to-delete demi"></span>
-    <p class="modal-message">{% ftlmsg 'modal-delete-warning-recovery-html' attrs='class="alias-to-delete"' email=request.user.email %}</p>
```

Comment:
Keep this `attrs` attribute and update the class name to be something like `js-set-relay-address`.

Reply:
Disregard this comment.